src/clang_complete.cc

```diff
@@ -425,10 +428,12 @@ void CompletionQueryMain(ClangCompleteManager* completion_manager) {
 
             ls_completion_item.documentation = ToString(
                 clang_getCompletionBriefComment(result.CompletionString));
-            ls_completion_item.sortText =
-                (const char)uint64_t(GetCompletionPriority(
+
+            buf << std::hex << std::setw(16) << std::setfill('0') <<
```

Comment:
Declare buf here. Does this cause any sort of perf impact?

Reply:
It probably would. Should we just write a tohex function ourselves?

Reply:
There is some perf logging in stderr, can you check the timings w/ and w/o? Global completion has the most candidates usually.

Reply:
Time to build 1799 completion results went from 3.3ms to 6ms

Reply:
Can we write the function ourselves then? That is a pretty large performance hit.

Reply:
How fast is sprintf(buf, "%016x", ...) ?